src/renderer/screens/DatabaseScreen/MainView.tsx

```diff
@@ -46,7 +67,9 @@ export default function MainView() {
           text: 'Close',
```

Comment:
There is 

```
{
  text: 'Close',
  disabled: tabs.length === 1,  // <-- disable can be use to prevent closing the last tab
  onClick: () => { /* ... */ }
}
```